### PR TITLE
edk update

### PR DESCRIPTION
**Pull request by giggsoff**
Update of `edk2` from stable201908 to stable202005 let me to start EVE on my Linux (qemu-system-x86_64 4.2.0) with `q35 machine` mode with `intel-iommu` device without displaying of UEFI SHELL.
But this requires additional verification on Mac OS and with EVE`s rootfs updates.

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>

### CODE REVIEW

File: pkg/uefi/Dockerfile [hunk 1]
```diff
@@ -19,9 +19,9 @@ FROM ${BUILD_CONTAINER} as build
 # hadolint ignore=DL3018
 RUN apk add --no-cache curl make gcc g++ python libuuid iasl nasm util-linux-dev bash git util-linux
 
-RUN git clone --depth 1 -b edk2-stable201908 https://github.com/tianocore/edk2.git /ws
+RUN git clone --depth 1 -b edk2-stable202005 https://github.com/tianocore/edk2.git /ws
 WORKDIR /ws
-RUN git submodule update --init --depth 1
+RUN git submodule update --init
```
Comment (rvs) [inline pkg/uefi/Dockerfile:24]: why did you remove --depth 1 @giggsoff ?

File: pkg/uefi/patch/000.patch [hunk 2]
```diff
@@ -1,25 +1 @@
...
       mXen = TRUE;
       return TRUE;
     }
+    if (!AsciiStrCmp ((CHAR8 *) Signature, "ZenZenZenZen")) {
+      mXen = TRUE;
+      return TRUE;
+    }
   }
 
   mXenLeaf = 0;
```
Comment (rvs) [inline pkg/uefi/patch/000.patch:1]: This doesn't seem right with all the ^M characters -- is this just a formatting issue of some kind?

File: pkg/uefi/Dockerfile (hunk 1 above)
Comment (giggsoff) [reply pkg/uefi/Dockerfile:24]: With `--depth 1` it fails with:
```error: Server does not allow request for unadvertised object 1cc9cde3448cdd2e000886a26acf1caac2db7cf1
Fetched in submodule path 'UnitTestFrameworkPkg/Library/CmockaLib/cmocka', but it did not contain 1cc9cde3448cdd2e000886a26acf1caac2db7cf1. Direct fetching of that commit failed.
The command '/bin/sh -c git submodule update --init --depth 1' returned a non-zero code: 1
exit status 1
```

So, I follow [the recommendation](https://github.com/tianocore/edk2#submodules) from git.

File: pkg/uefi/patch/000.patch (hunk 2 above)
Comment (giggsoff) [reply pkg/uefi/patch/000.patch]: Sorry, @rvs, it is my mistake. File has `CRLF` line endings, so patch must have mixed `LF` and `CRLF` line endings. `core.autocrlf` of git broke my edits.
Now it is fixed.
